internal/mtls/mtls.go

```diff
@@ -0,0 +1,130 @@
+package mtls
```

Comment:
1. Add the missing header.
1. This package is very specific to Hubble. So rename the package to either `hubble` (my preference) or `hubble_mtls`. If renaming into `hubble`, it would be a good opportunity to move a bunch of code out of the `controller` package into this one, as `controller` is getting bloated.
```suggestion
// Copyright 2025 Illumio, Inc. All Rights Reserved.

package hubble
```